packages/babel-parser/src/tokenizer/index.js

```diff
@@ -897,6 +897,19 @@ export default class Tokenizer extends LocationParser {
 
     let total = 0;
 
+    // Called from readHexChar
+    if (radix === 16) {
+      const unicode = this.input.slice(this.state.pos, this.state.pos + len);
+
+      if (unicode.includes("_")) {
+        const numSeparatorPos = this.input.indexOf("_");
+        this.raise(
```

Comment:
I don't think we should throw an error like this if user does _not_ enable `numericSeparator` parser plugin.

Reply:
You're right.